src/com/ar4android/AR4AndroidApplication.java

```diff
@@ -1,83 +1,157 @@
 package com.ar4android;
 
-
-
-import android.util.Log;
-
+import com.jme3.animation.AnimChannel;
+import com.jme3.animation.AnimControl;
+import com.jme3.animation.AnimEventListener;
+import com.jme3.animation.LoopMode;
 import com.jme3.app.SimpleApplication;
-
+import com.jme3.light.DirectionalLight;
 import com.jme3.material.Material;
+import com.jme3.math.ColorRGBA;
+import com.jme3.math.FastMath;
+import com.jme3.math.Quaternion;
+import com.jme3.math.Vector3f;
 import com.jme3.renderer.Camera;
+import com.jme3.renderer.RenderManager;
 import com.jme3.renderer.ViewPort;
 import com.jme3.scene.Geometry;
+import com.jme3.scene.Spatial;
 import com.jme3.scene.shape.Quad;
 import com.jme3.texture.Image;
 import com.jme3.texture.Texture2D;
 
-public class AR4AndroidApplication extends SimpleApplication {
+public class AR4AndroidApplication extends SimpleApplication  implements AnimEventListener { 
 
-	private static final String TAG = "CameraAccessJME";
+	private static final String TAG = "SuperimposeJME";
+	
 	// The geometry which will represent the video background
 	private Geometry mVideoBGGeom;
 	// The material which will be applied to the video background geometry.
 	private Material mvideoBGMat;
 	// The texture displaying the Android camera preview frames.
 	private Texture2D mCameraTexture;
-	// the JME image which serves as intermediate storage place for the Android camera frame before the pixels get uploaded into the texture.
+	// the JME image which serves as intermediate storage place for the Android
+	// camera frame before the pixels get uploaded into the texture.
 	private Image mCameraImage;
 	// A flag indicating if the scene has been already initialized.
 	private boolean mSceneInitialized = false;
 	// A flag indicating if a new Android camera image is available.
-	boolean	mNewCameraFrameAvailable = false;
+	boolean mNewCameraFrameAvailable = false;
 
+	private float mForegroundCamFOVY = 50; // for a Samsung Galaxy SII
+	
+	// for animation	
+	// The controller allows access to the animation sequences of the model
+	private AnimControl mAniControl;
+	// the channel is used to run one animation sequence at a time
+	private AnimChannel mAniChannel;
+	
+  
 	public static void main(String[] args) {
 		AR4AndroidApplication app = new AR4AndroidApplication();
 		app.start();
 	}
 
-	// The default method used to initialize your JME application.
 	@Override
 	public void simpleInitApp() {
-		// Do not display statistics or frames per second
+		// Do not display statistics
 		setDisplayStatView(false);
 		setDisplayFps(false);
-		// We use custom viewports - so the main viewport does not need to contain the rootNode
+		// we use our custom viewports - so the main viewport does not need the  rootNode
 		viewPort.detachScene(rootNode);
 		initVideoBackground(settings.getWidth(), settings.getHeight());
+		initForegroundScene();	
```

Comment:
Note that the order in which the camera and viewports are initialized is important.